### PR TITLE
Update for Kubernetes v1.18.0

### PR DESCRIPTION
**Pull request by mjura**
It is adding Kubernetes v1.18 and all necessary containers to skuba.

It depends on: https://github.com/SUSE/skuba/pull/1024

### CODE REVIEW

File: internal/pkg/skuba/kubernetes/versions.go [hunk 1]
```diff
@@ -81,6 +81,27 @@ type ClusterAddonsKnownVersions = func(clusterVersion *version.Version) AddonsVe
 
 var (
 	supportedVersions = KubernetesVersions{
+    "1.18.0": KubernetesVersion{
```
Comment (evrardjp) [inline internal/pkg/skuba/kubernetes/versions.go]: indentation seems off.

File: ci/infra/openstack/cpi.auto.tfvars [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Enable CPI integration with OpenStack
-#cpi_enable = true
+# cpi_enable = true
```
Comment (evrardjp) [inline ci/infra/openstack/cpi.auto.tfvars]: Not sure this change needs to be in ;)

File: internal/pkg/skuba/kubernetes/versions.go (hunk 1 above)
Comment (mjura) [reply internal/pkg/skuba/kubernetes/versions.go]: `go fmt`

**Comment by mjura** [comment]
It will be continued in https://github.com/SUSE/skuba/pull/1028

**Pull request closed without merging**